api/tests/test_views.py

```diff
@@ -126,19 +126,19 @@ def test_can_post_lesson(self):
               'answers': [
                   {
                     'answer': 'answer1 description',
-                    'correct': False
+                    'correct': 'false'
                   },
                   {
                     'answer': 'answer2 description',
-                    'correct': True
+                    'correct': 'true'
                   },
                   {
                     'answer': 'answer3 description',
-                    'correct': False
+                    'correct': 'false'
                   },
                   {
                     'answer': 'answer4 description',
-                    'correct': False
+                    'correct': 'false'
                   }
                 ]
```

Comment:
We will need to find out why Python uses booleans in this way and how to work with it in the future.